cli.js

```diff
@@ -28,6 +29,12 @@ const cli = meow(`
 
 updateNotifier({pkg: cli.pkg}).notify();
 
+const pkg = readPkgUp.sync().pkg;
+if (!pkg) {
+	console.error('\nNo package.json found, are you in the correct project?');
```

Comment:
Why the newline? Just wondering. Maybe add `[log-symbols](https://github.com/sindresorhus/log-symbols).error` in front of the error message. I'm also not sure about the message itself, can't come up with something better though :).